src/depackers/xz.h

```diff
@@ -241,7 +241,27 @@ XZ_EXTERN void xz_dec_end(struct xz_dec *s);
  * care about the functions below.
  */
 #ifndef XZ_INTERNAL_CRC32
+#	ifdef __KERNEL__
+#		define XZ_INTERNAL_CRC32 0
+#	else
 #		define XZ_INTERNAL_CRC32 1
+#	endif
+#endif
+
+/*
+ * If CRC64 support has been enabled with XZ_USE_CRC64, a CRC64
+ * implementation is needed too.
+ */
+#ifndef XZ_USE_CRC64
+#	undef XZ_INTERNAL_CRC64
+#	define XZ_INTERNAL_CRC64 0
+#endif
+#ifndef XZ_INTERNAL_CRC64
+#	ifdef __KERNEL__
```

Comment:
same

Reply:
This one wasn't even there in the last version of XZ Embedded.

Reply:
I removed the `ifdef __KERNEL__` branch of this but left it otherwise, since "theoretically" CRC-64 might be useful if we end up with multiple formats that need it.